README.md

```diff
@@ -0,0 +1,26 @@
+# Sawtooth PoET
+
+Sawtooth PoET is a consensus engine for Hyperledger Sawtooth based on a
+solution to Byzantine Generals Problem that utilizes
+a “trusted execution environment(TEE)”.
+
+The PoET specification is available in
+[sawtooth documentation]( https://sawtooth.hyperledger.org/docs/core/nightly/master/architecture/poet.html)
+
+Currently, Sawtooth PoET implements the non-sgx simulator mode and is
+production worthy.
```

Comment:
Claims about appropriate environments such as "production" have been avoided because they have different meanings to different people. Since PoET/CFT is not BFT, there are a large number of production use cases where PoET/CFT isn't appropriate, for example.

Reply:
Here's some text I was working on for the release notes: 
```Sawtooth PoET is deployable as a pure python application using a simulated enclave (poet-simulator) or with a c++ module implementing an SGX enclave (poet-sgx).
Poet-simulator has been ported to the Consensus Engine interface newly released in Sawtooth v1.1.
Poet-sgx has not been validated on Sawtooth v1.1.
Users relying on poet-sgx are recommended to remain on Sawtooth v1.0.5.
Users relying on poet-simulator are recommended to upgrade to Sawtooth v1.1
We are working on a new implementation of PoET and its TEE enclave. It is anticipated for a point release in the near future.```